app/src/main/res/values/strings.xml

```diff
@@ -6,19 +6,44 @@
     <string name="error_message_field_empty">Field cannot be empty</string>
     <string name="error_message_logout_failure">Error when during logout process</string>
     <string name="confirmation_message_saved">Saved</string>
+    <string name="button_cancel">Cancel</string>
+    <string name="button_ok">OK</string>
```

Comment:
IIRC, "OK" and "Cancel" are already defined in `android.R.string.*`, maybe you want to use those, they are translated too.

Reply:
Hmmmm... Not my strings - I just reorganised them by "scopes"... Could not find where these were used, though, so I'm removing them. Thanks!!